platforms/corda-ent/charts/signer/values.yaml

```diff
@@ -131,38 +121,65 @@ serviceLocations:
 # Provide signers variables that will go to the template file
 # Like the time interval for checking different tasks
 signers:
-  # For checking certificate signing request schedule
+  # For checking Certificate Signing Request (CSR) schedule
   CSR:
     schedule:
-      # Ex interval: 1m
+      # Eg. interval: 1m
       interval:
-  # For checking certificate revocation list schedule
+  # For checking Certificate Revocation List (CRL) schedule
   CRL:
     schedule:
-      # Ex interval: 1d
+      # Eg. interval: 1d
       interval:
+  # For checking with NetworkMap (NMS)
   NetworkMap:
-    # schedule for checking with nms
     schedule:
-      # Ex interval: 1d
+      # Eg. interval: 1d
       interval:
   # For checking network parameters interval
   NetworkParameters:
     schedule:
-      # Ex interval: 1m
+      # Eg. interval: 1m
       interval:
 
 
-# Provide the JVM for jar
-# cordaJarMx: 1
-cordaJarMx:
-
-healthCheckNodePort:
+#############################################################
+#                       Settings                            #
+#############################################################
+# Provide volume related specifications
+volume:
```

Comment:
same comments as idman

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5); all values have been grouped under common key `config`